docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,112 @@
+---
+title: High availability of Aiven for MySQL®
+sidebar_label: High availability
+---
+
+Aiven for MySQL® is available on a variety of plans, offering different levels of high availability. The selected plan defines the features available, and a summary is provided in the table below:
+
+| Plan         | High availability features                                      | Backup history | Uptime  |
+| ------------ | --------------------------------------------------------------- | -------------- | ------- |
+| **Hobbyist** | Single node with limited availability                           | 2 days         | 98.5 %  |
+| **Startup**  | Single node with limited availability                           | 2 days         | 99 %    |
+| **Business** | Two nodes (primary + standby: basic high availability)          | 14 days        | 99.9 %  |
+| **Premium**  | Three nodes (primary + 2 x standby: stronger high availability) | 30-day         | 99.99 % |
+
+## About primary and standby nodes
+
+Aiven's Business and Premium plans offer primary nodes and standby nodes. A standby
+service is useful for multiple reasons:
+
+-   Provides another physical copy of the data in case of hardware,
+    software, or network failures
+-   Typically reduces the data loss window in disaster scenarios
+-   Provides a quicker database time to restore with a controlled
+    failover in case of failures, as the standby is already installed,
+    running, and synchronised with the data
+-   Can be used for read-only queries to reduce the load on the primary
```

Comment:
Care with this: sending reads on a standby node might be appealing, but might not work well when a failure happens:
- when the primary fails, failing-over to the standby might increase load and reduce performance,
- when a standby fails, the reads sent to the other standby (or the primary) can cause similar as above (or the reads on a standby might start failing).

So I would not advertise standby as 100% reliable and available "read-only" replicas.  This is another service.  But standbys can be used for low-availability or occasionnal read-only queries.